mypyc/primitives/registry.py

```diff
@@ -58,6 +67,8 @@
 # Primitive ops for reading module attributes (key is name such as 'builtins.None')
 name_ref_ops = {}  # type: Dict[str, OpDescription]
 
+call_c_ops = {}  # type: Dict[str, List[CFunctionDescription]]
```

Comment:
Since we store a method call here, the name is the short name of a method (i.e, not a fully qualified name such as `builtins.ord`). For consistency with the existing registries, a better name would be `c_method_call_ops` or similar. We can then have a separate dictionary for module-level functions.